### ISSUE TITLE
[Suggestion] Overgrown Style: Implement "Broken Glass" inside the Laser Emiter and Relay Sockets

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#649]
They have got an awesome overgrown look, but adding a final touch like a "Broken glass" on them would be cool.

![20160729214351_1](https://cloud.githubusercontent.com/assets/12244376/17267669/4e3f536a-55e5-11e6-965d-836dc1e9aa14.jpg)

**Comment by ZombiePwner1** [comment]
Adding broken glass on them could be abit tricky.

**Comment by Konclan** [comment]
Is this even a good idea? even in overgrown not everything has to be cracked, just seems unnecessary

**Comment by Luke18033** [comment]
We might want to do it for the glass covering the floor laser emitter, but for everything else it's not really necessary.

**Comment by LautaroL20** [comment]
On emmiters and relays, since actually both uses glass. It doesnt make sense to only add crakered glass on emmiter but not on relays.

**Comment by Luke18033** [comment]
Only the emitter *on the floor*, which is embedded slightly further with glass covering it. The emitter itself wouldn't use a cracked model. Really though even that's unnecessary, just the dirty versions we have now are good enough.

**Comment by ZombiePwner1** [comment]
I believe this already been done?

**Comment by LautaroL20** [comment]
No, it is not implemented. Do not close an issue based on your faith, wheels.

**Comment by ZombiePwner1** [comment]
@LautaroL20 Says the wheely.
